packages/backend-common/package.json

```diff
@@ -42,6 +42,7 @@
     "@backstage/types": "^1.0.0",
     "@google-cloud/storage": "^6.0.0",
     "@keyv/redis": "^2.2.3",
+    "@kubernetes/client-node": "^0.17.0",
```

Comment:
I'm not super happy about introducing this dependency here tbh, It's pretty large and we're currently having some trouble with it because of slow releases. In the end I think it's likely alright for now but as we head towards #11611 it likely makes sense to break this implementation into a separate package.

Reply:
should I keep it for the moment?

Reply:
Yep let's keep it and request for now